code/controllers/configuration/sections/event_configuration.dm

```diff
@@ -58,4 +61,6 @@
 			ASSERT(target in list(EVENT_LEVEL_MUNDANE, EVENT_LEVEL_MODERATE, EVENT_LEVEL_MAJOR))
 			first_run_times[target] = list("lower" = assoclist["lower_bound"] MINUTES, "upper" = assoclist["upper_bound"] MINUTES)
 
+	if(isnum(data["blob_highpop_trigger"]))
+		CONFIG_LOAD_NUM(blob_highpop_trigger, data["blob_highpop_trigger"])
```

Comment:
```suggestion
	CONFIG_LOAD_NUM(blob_highpop_trigger, data["blob_highpop_trigger"])
```

```dm
/// Wrapper to not overwrite a variable if a list key doesnt exist. Auto casts to number.
#define CONFIG_LOAD_NUM(target, input) \
	if(!isnull(input)) {\
		target = text2num(input)\
	}
```

We already have this stuff in place, hence why none of the other wrappers have it.